pygments/lexers/parsers.py

```diff
@@ -534,7 +529,7 @@ def analyse_text(text):
 #    filenames = ['*.G', '*.g']
 #
 #    def __init__(self, **options):
-#        super(AntlrCLexer, self).__init__(CLexer, AntlrLexer, **options)
+#        super().__init__(CLexer, AntlrLexer, **options)
```

Comment:
Given this is all commented out, maybe better get rid of this wholesale?

Reply:
Yeah, sure.